packages/playback-core/src/index.ts

```diff
@@ -667,22 +674,28 @@ export const getDRMConfig = (
 
 export const getAppCertificate = async (appCertificateUrl: string) => {
   const resp = await fetch(appCertificateUrl);
+  if (resp.status !== 200) {
+    return Promise.reject(resp);
+  }
   const body = await resp.arrayBuffer();
   return body;
 };
 
 export const getLicenseKey = async (message: ArrayBuffer, licenseServerUrl: string) => {
-  const licenseResponse = await fetch(licenseServerUrl, {
+  const resp = await fetch(licenseServerUrl, {
     method: 'POST',
     headers: { 'Content-type': 'application/octet-stream' },
     body: message,
   });
-  const keyBuffer = await licenseResponse.arrayBuffer();
+  if (resp.status !== 200) {
+    return Promise.reject(resp);
+  }
+  const keyBuffer = await resp.arrayBuffer();
   return new Uint8Array(keyBuffer);
 };
 
 export const setupNativeFairplayDRM = (
-  props: Partial<Pick<MuxMediaPropsInternal, 'playbackId' | 'drmToken' | 'customDomain' | 'drmTypeCb'>>,
+  props: Partial<Pick<MuxMediaPropsInternal, 'playbackId' | 'tokens' | 'playbackToken' | 'customDomain' | 'drmTypeCb'>>,
```

Comment:
NOTE: There's a lot of very similar code for error tracking and translation, both here and with the corresponding hls.js function (below). I couldn't think of any straightforward way to abstract this without even larger refactors, but if anything jumps out, even if minor improvements, I'm definitely amenable.